packages/components/addon/components/hds/text/body.hbs

```diff
@@ -0,0 +1,6 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Hds::Text @group="body" @size={{this.size}} @weight={{this.weight}} @tag={{@tag}} ...attributes>{{yield}}</Hds::Text>
```

Comment:
a small nit pick, `@size` attribute might be a little confusing naming, would `@variant` be better

Reply:
@aklkv here's my thinking
- until now we have always avoided using `@variant` as argument for our components, **on purpose** because it's an heavy-loaded term (in Figma a component "variant" has a specific meaning) and could lead to confusion between designers and developers, and I would prefer to continue to follow this "rule"
- in general what we're referring with `100/200/300/400/500` is the **sizing** scale for that particular style (eg. `Display`) so `@size` is the thing that mostly aligns with that semantic

Also, this specific code is internal, not external-facing. Consumers will always write something like this
```
<Hds::Text::Display @size="300" ...>
```
which I personally don't think it's too confusing: I don't see them writing something like this
```
<Hds::Text::100 @size="Display" ...>
```
or 
```
<Hds::Text::Display @size="display" ...>
```

Reply:
right ok, this makes sense 👍🏻